Slider/Slider.js

```diff
@@ -267,60 +89,271 @@ const SliderBase = kind({
-		tooltip: ({tooltip}) => tooltip === true ? ProgressBarTooltip : tooltip
-	},
-
-	render: ({css, disabled, focused, tooltip, ...rest}) => {
-		delete rest.activateOnSelect;
-		delete rest.active;
-		delete rest.onActivate;
-		delete rest.knobStep;
-		delete rest.showAnchor;
-
-		return (
-			<UiSlider
-				{...rest}
-				aria-disabled={disabled}
-				css={css}
-				disabled={disabled}
-				progressBarComponent={
-					<ProgressBar css={css} />
-				}
-				tooltipComponent={
-					<ComponentOverride
-						component={tooltip}
-						css={css}
-						visible={focused}
-					/>
-				}
-			/>
-		);
+	}, props, spotlightAccelerator);
+
+
+	// if the props includes a css map, merge them together
+	let mergedCss = componentCss;
+	if (css) {
+		const allowedClassNames = Object.keys(componentCss);
+		mergedCss = mergeClassNameMaps(componentCss, css, allowedClassNames);
 	}
-});
+
+	const componentClassName = classnames(
+		componentCss.slider,
+		className,
+		{
+			[componentCss.active]: active,
+			[componentCss.showAnchor]: showAnchor
+		},
+		css && css.slider
+	);
+
+	useEffect(() => {
+		spotlightAccelerator.current = new Accelerator(keyFrequency);
+	}, [keyFrequency]);
+
+	delete rest.activateOnSelect;
+	delete rest.knobStep;
+	delete rest.onActivate;
+	delete rest.step;
+	delete rest.tooltip;
+
+	return (
+		<UiSlider
+			{...rest}
+			{...handlers}
+			aria-disabled={disabled}
+			className={componentClassName}
+			css={mergedCss}
+			disabled={disabled}
+			progressBarComponent={
+				<ProgressBar css={mergedCss} />
+			}
+			step={step}
+			tooltipComponent={
+				<ComponentOverride
+					component={tooltip}
+					css={mergedCss}
+					visible={focused}
+				/>
+			}
+		/>
+	);
+};
+
+SliderBase.displayName = 'Slider';
+
+SliderBase.propTypes = /** @lends sandstone/Slider.SliderBase.prototype */ {
+	/**
+	 * Activates the component when selected so that it may be manipulated via the directional
+	 * input keys.
+	 *
+	 * @type {Boolean}
+	 * @public
+	 */
+	activateOnSelect: PropTypes.bool,
+
+	/**
+	 * Sets the knob to selected state and allows it to move via 5-way controls.
+	 *
+	 * @type {Boolean}
+	 * @public
+	 */
+	active: PropTypes.bool,
+
+	/**
+	 * Customizes the component by mapping the supplied collection of CSS class names to the
+	 * corresponding internal elements and states of this component.
+	 *
+	 * The following classes are supported:
+	 *
+	 * * `slider` - The root component class
+	 *
+	 * @type {Object}
+	 * @public
+	 */
+	css: PropTypes.object,
+
+	/**
+	 * Disables component and does not generate events.
+	 *
+	 * @type {Boolean}
+	 * @public
+	 */
+	disabled: PropTypes.bool,
+
+	/**
+	 * Indicates that the slider has gained focus and if the tooltip is present, it will be
+	 * shown.
+	 *
+	 * @type {Boolean}
+	 * @public
+	 */
+	focused: PropTypes.bool,
+
+	/**
+	 * Controls the keydown frequency with which the acceleration will "freeze".
+	 * While frozen, the value of the slider is not changed via arrow key.
+	 *
```

Comment:
Can we add some examples for using this prop?
It's not easy to grasp the meaning of the value here before actually using it.

Reply:
Good, I added example likes tooltip prop

Reply:
Would you add some explanation of what the numbers mean? like.. A higher number, slower speed.